components/menu/OverflowMenu/stories.react.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import classNames from 'classnames/bind'
+import OverflowMenu from './react'
+import Button from '../../forms/button/Button/react'
+import storyStyles from './stories-style.module.scss'
+
+const cx = classNames.bind(storyStyles)
+
+storiesOf('Menu/Overflow Menu', module)
+  .add(
+    'default',
+    () => (
+      <OverflowMenu>
+        <p>SAVED ON: 05/30/18<br />
+          EDITED BY: LOREM IPSUM
+        </p>
```

Comment:
While this has no effect on the component whatsoever, I think we should remove all caps-lock text as some people may use existing code as a reference. We should be using `text-transform: uppercase` for caps

Reply:
I went ahead and did as you requested, though I feel the decision to use caps lock vs the text-transform is something to be evaluated on a case-by-case basis. Specifically, if the design calls for uppercase, but the data is coming in dynamically then I can see the need for text transforming it. If the data is static, then personally, I I feel it is ideal to have it caps locked.

Reply:
Design and aesthetics removed, capital letters are for acronyms. If Google crawls one of our pages, a screen reader would not read MASONITE as Masonite, but M-A-S-O-N-I-T-E. The text we had capitalized was purely visual.